File: Java/src/apuntes/op-logicos.java
import java.util.*;

class Main
{ 
    public static void main(String args[])
  { 
    // Los operadores son &&, || y !
    // Los resultados de combinar TRUE y FALSE con estos operadores son:
    
    // Tabla de resultados operador && ("y")
    // true && true = true
    boolean resultado = true && true;
    System.out.println(resultado);

    // true && false = false
    boolean resultado = true && false;
    System.out.println(resultado);

    // false && true = false
    boolean resultado = false && true;
    System.out.println(resultado);

    // false && false = false
    boolean resultado = false && false;
    System.out.println(resultado);


    // Tabla de resultados operador || ("o")
    // true || true = true
    boolean resultado = true || true;
    System.out.println(resultado);
    // true || false = true
    boolean resultado = true || false;
    System.out.println(resultado);
    // false || true = true
    boolean resultado = false || true;
    System.out.println(resultado);
    // false || false = false
    boolean resultado = false || false;
    System.out.println(resultado);


    // Operador ! 
    // !true = false
    boolean resultado = !true;
    System.out.println(resultado);

    // !false = true
    boolean resultado = !false;
    System.out.println(resultado);

  }
}

File: Lua/src/apuntes/op-logicos.lua
-- Ejemplos de operadores lógicos

-- Los operadores son AND, OR y NOT
-- Los resultados de combinar TRUE y FALSE con estos operadores son:


-- Tabla de resultados operador AND ("y")

-- true and true = true
local resultado = true and true
print(resultado)

-- true and false = false
local resultado = true and false
print(resultado)

-- false and true = false
local resultado = false and true
print(resultado)

-- false and false = false
local resultado = false and false
print(resultado)


-- Tabla de resultados operador OR ("o")

-- true or true = true
local resultado = true or true
print(resultado)

-- true or false = true
local resultado = true or false
print(resultado)

-- false or true = true
local resultado = false or true
print(resultado) 

-- false or false = false
local resultado = false or false
print(resultado)


-- Operador NOT

-- not true = false
local resultado = not true
print(resultado)

-- not false = true
local resultado = not false
print(resultado)